the-blue-alliance-ios/View Elements/Match/MatchSummaryView.swift

```diff
@@ -14,12 +14,23 @@ class MatchSummaryView: UIView {
             configureView()
         }
     }
-    
+
     // change this variable so that the teams are shown as buttons
     private var teamsTappable: Bool = false
+    var calculatedHeight: CGFloat {
+        let multiplier: CGFloat = 2.5 // change this value to add vertical padding
+        return (currentFontSize - 14) * multiplier + 90
+    }
 
-    private let winnerFont = UIFont.systemFont(ofSize: 14, weight: UIFont.Weight.bold)
-    private let notWinnerFont = UIFont.systemFont(ofSize: 14, weight: UIFont.Weight.medium)
+    private var currentFontSize: CGFloat {
+        return teamsTappable ? 18 : 14 // change this value to change button size
+    }
+    private var winnerFont: UIFont {
+        return UIFont.systemFont(ofSize: currentFontSize, weight: UIFont.Weight.bold)
```

Comment:
Can we see if we can get some dynamic type fonts in here? I can help pick some, if we need.